packages/dds/migration-shim/src/types.ts

```diff
@@ -37,6 +37,7 @@ export interface IShimDeltaHandler extends IDeltaHandler {
  * An interface for interrogating ops to see if they are stamped or not so that we can choose to drop them.
  */
 export interface IStampedContents {
+	type: string;
 	fluidMigrationStamp?: IChannelAttributes;
```

Comment:
This property should be documented, in particular covering:
1. What does it mean if omitted
2. What do we expect the attributes to be in normal usage